cmd/internal/types.go

```diff
@@ -160,6 +161,8 @@ func parseValue(val sqltypes.Value, columnType string) sqltypes.Value {
 	} else if strings.HasPrefix(columnType, "set") {
 		values := parseEnumOrSetValues(columnType)
 		return mapSetValue(val, values)
+	} else if lowerCased := strings.ToLower(columnType); strings.HasPrefix(lowerCased, "time") || strings.HasPrefix(lowerCased, "date") {
```

Comment:
Do we not have an actual type here? If not, then using the MySQL column type string that we include works.

Reply:
![CleanShot 2025-02-18 at 08 36 39@2x](https://github.com/user-attachments/assets/421dcb57-7512-4a58-a9e4-b1670415b42f)

Do you mean these methods? I can use them but wondering how to handle `date`?

Reply:
note: we talked offline and I modified the code to use `Type`!